recipe/sysconfigdata/default/_sysconfigdata_linux.py

```diff
@@ -641,7 +641,7 @@
  'SIZEOF_VOID_P': 8,
  'SIZEOF_WCHAR_T': 4,
  'SIZEOF__BOOL': 1,
- 'SOABI': 'cpython-@PYVERNODOTS@m-@ARCH@-linux-gnu',
+ 'SOABI': 'cpython-@PYVERNODOTS@-@ARCH@-linux-gnu',
```

Comment:
@jjhelmus does that mean the `m` should be removed everywhere?
If so:
```sh
grep -rnE '(3\.?[7-8]|@)m'                                                                                                                                                                                                           
sysconfigdata/default/_sysconfigdata_osx.py:13: 'BLDLIBRARY': '-L. -lpython@PYVER@m',
sysconfigdata/default/_sysconfigdata_osx.py:52: 'CONFINCLUDEPY': '/opt/anaconda1anaconda2anaconda3/include/python@PYVER@m',
sysconfigdata/default/_sysconfigdata_osx.py:91: 'EXT_SUFFIX': '.cpython-37m-darwin.so',
sysconfigdata/default/_sysconfigdata_osx.py:482:                   '/opt/anaconda1anaconda2anaconda3/include/python@PYVER@m '
sysconfigdata/default/_sysconfigdata_osx.py:483:                   '/opt/anaconda1anaconda2anaconda3/include/python@PYVER@m',
sysconfigdata/default/_sysconfigdata_osx.py:485: 'INCLUDEPY': '/opt/anaconda1anaconda2anaconda3/include/python@PYVER@m',
sysconfigdata/default/_sysconfigdata_osx.py:491: 'INSTSONAME': 'libpython@PYVER@m.dylib',
sysconfigdata/default/_sysconfigdata_osx.py:500: 'LDLIBRARY': 'libpython@PYVER@m.dylib',
sysconfigdata/default/_sysconfigdata_osx.py:507: 'LDVERSION': '@PYVER@m',
sysconfigdata/default/_sysconfigdata_osx.py:516: 'LIBPL': '/opt/anaconda1anaconda2anaconda3/lib/python@PYVER@/config-@PYVER@m-darwin',
sysconfigdata/default/_sysconfigdata_osx.py:517: 'LIBRARY': 'libpython@PYVER@m.a',
sysconfigdata/default/_sysconfigdata_osx.py:655: 'SOABI': 'cpython-37m-darwin',
sysconfigdata/default/_sysconfigdata_linux.py:13: 'BLDLIBRARY': '-L. -lpython@PYVER@m',
sysconfigdata/default/_sysconfigdata_linux.py:87: 'EXT_SUFFIX': '.cpython-@PYVERNODOTS@m-@ARCH@-linux-gnu.so',
```
(... I'm just a "Jon Snow" coincidentally passing  by and "know nothing" of what y'all doing here :man_shrugging:)

Reply:
On linux yes, I need to check for macOS

Reply:
Yup, macOS as well

Reply:
As a bit of background.  The default _sysconfigdata files are used when Python builds an extension outside of conda build, for example if you run `python setup.py build` or similar in a directory with a Python library with a compiled extension. 

In these cases the system compilers are (most likely) used so the compiler commands and flags that were used to build Python itself which come from the "Anaconda" compilers are likely not valid.  The default sysconfigdata files provide a generic set of compiler commands and flags that should work in most cases.  

These can be overridden by setting `_PYTHON_SYSCONFIGDATA_NAME` or `_CONDA_PYTHON_SYSCONFIGDATA_NAME` to point to a different sysconfigdata file.  The compiler activation scripts set the later which is why the correct commands and flags are passed in conda-build.